.nais/naiserator-prod.yaml

```diff
@@ -8,8 +8,8 @@ metadata:
 spec:
   image: {{ image }}
   replicas:
-    min: 2
-    max: 4
+    min: 1
+    max: 1
```

Comment:
Todo: Vi må huske å rette opp i dette når vi får er ferdige med å forbedre feilhåndteringa.

Reply:
Hvis vi skal takle duplikater som kommer med omtrent samtidig så må kanskje dette være den permanente løsningen for å sikre sekvensiell prosessering.